### PR TITLE
CIV-14703 Dashboard notification and status accept FullAdmit PayBySetDate

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14703

### Change description ###
Set up dashboard status for FullAdmit PayBySetDate

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java [hunk 1]
```diff
@@ -90,6 +92,12 @@ public String getScenario(CaseData caseData) {
             }
         } else if (isClaimantRejectRepaymentPlan(caseData)) {
             return SCENARIO_AAA6_CLAIMANT_INTENT_REJECT_REPAYMENT_ORG_LTD_CO_DEFENDANT.getScenario();
+        } else if (
+            !featureToggleService.isJudgmentOnlineLive()
+                && !NO.equals(caseData.getApplicant1Represented())
```
Comment (paul-pearson) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java:97]: I think you can change this to
`&& caseData.isApplicantLiP()`

Comment (kenneth-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java]: Updated from review

**Review by manish14836 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java [hunk 2]
```diff
@@ -90,6 +91,12 @@ public String getScenario(CaseData caseData) {
             }
         } else if (isClaimantRejectRepaymentPlan(caseData)) {
             return SCENARIO_AAA6_CLAIMANT_INTENT_REJECT_REPAYMENT_ORG_LTD_CO_DEFENDANT.getScenario();
+        } else if (
+            !featureToggleService.isJudgmentOnlineLive()
+                && !caseData.isApplicantLiP()
```
Comment (portizdev) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java:96]: Should it be caseData.isApplicantLiP()? You're checking if applicant is LR

Comment (kenneth-hmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java:96]: It is checking if applicant is LR or not. If applicant is LR the Applicant1Represented field can be null or set as YES

**Review by portizdev — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Pull request merged**